src/sections/Community/Handbook/mentorships.js

```diff
@@ -96,73 +74,137 @@ const MentorshipPrograms= () => {
       <TOC />
       <ConductWrapper>
         <Container>
-          <p className="page-main-para">As an open-source organization and a community, we participate in different mentorships programs where project ideas are submitted for people to contribute to during the program. </p>
-        </Container>
-        <div className="showcase">
-          <Slider
-            className="slider"
-            arrows={true}
-            infinite={false}
-            speed="500"
-            slidesToScroll={1}
-          > 
-            <div className="box">  
-              <h2>Google Summer of Code</h2>
-              <img className="poster" src={GSOC}/>
-              <p>Google Summer of Code is a global program focused on bringing more student developers into open source software development. Students work with an open-source organization on a 10-week programming project during their break from school.</p>
-              <p>Every Summer, Layer5 submits a new project idea for student developers to contribute to for a particular period. Also, stipends are awarded at the end of the program to the contributors.</p>
-              <p>The critical component of these projects is our Community. This community, which you will join as a participant in Google Summer of Code, improves the world of diverse cloud-native systems. Your contributions will affect people you've never met. The Layer5 community includes software engineers, researchers, students, artists, system administrators, operators, and web designers, all of whom will be happy to help you get started.</p>
-              <p>We believe that all contributors should expect and be part of a safe and friendly environment for constructive contribution. We can more effectively and successfully compare and challenge different ideas to find the best solutions for advancement while building our community’s size, diversity, and strength.</p>
-              <p>Learn more about the program and how to participate <a href="https://summerofcode.withgoogle.com/">here</a></p>
+        <div className="explain-2">
```

Comment:
Let's have a data file that contains the data for all the programs such as description, button link, image link, and then import that data file and traverse it to show the data. It will help organize things in the long run.